instaloader/structures.py

```diff
@@ -43,18 +43,68 @@ class PostCommentAnswer(NamedTuple):
 PostCommentAnswer.likes_count.__doc__ = "Number of likes on comment."
 
 
-class PostComment(NamedTuple):
-    id: int
-    created_at_utc: datetime
-    text: str
-    owner: 'Profile'
-    likes_count: int
-    answers: Iterator[PostCommentAnswer]
+class PostComment:
+    def __init__(self, node: Dict[str, Any], answers: Iterator['PostCommentAnswer'],
+                 context: 'InstaloaderContext', shortcode: str):
+        self._node = node
+        self._answers = answers
+        self._context = context
+        self.shortcode = shortcode
```

Comment:
We should pass a `Post` instance rather than just the shortcode, this seems more future-proof in case we once need to access other fields of the Post.

We should also prefix the field with an underscore, to avoid giving the impression that it can be changed from outside.

Reply:
> I'm not very sure what do you mean by passing `Post` instance? we can pass its properties and such but itself?

Yes, we can pass a Post instance.

```python
            return PostComment(context=self._context, node=node,
                               answers=_postcommentanswers(node), post=self)
```

Reply:
Hello, Sorry I missed the release v4.10. i've been quiet offline for the past weeks.
It should be good to merge now.